docs/gettingstarted.md

```diff
@@ -29,34 +33,28 @@ quaca/build $ make quaca
 
 An executable called `quaca` should now have been build and can be found in the `quaca/bin` directory.
 
-## Documentation
-The documentation can either be seen [here](https://myoelmy.github.io/quaca) or it can be viewed locally using the following command from the command line
-### Python 2.7
-```bash
-quaca $ cd docs && python -m SimpleHTTPServer 3000
+### Documentation
+The documentation can either be seen [here](https://myoelmy.github.io/quaca) or be viewed locally using the following command from the command line
 ```
-### Python 3
```

Comment:
I don't think that it is a good idea to only leave the Python 2.7 solution as this version of Python is no longer development.

Reply:
Sorry, this was not intended. I'll fix it.